test/Microsoft.AspNetCore.Razor.Evolution.Test/IntegrationTests/RazorIRNodeWriter.cs

```diff
@@ -147,7 +147,10 @@ protected void WriteName(RazorIRNode node)
 
         protected void WriteLocation(RazorIRNode node)
         {
-            _writer.Write(node.SourceLocation.ToString());
+            if (node.DocumentLocation != null)
```

Comment:
This is still just legacy for now, we haven't yet written the location/range abstractions we're going to use.